README.rst

```diff
@@ -1,22 +1,24 @@
 h
 =
 
-.. image:: https://travis-ci.org/hypothesis/h.svg?branch=master
-   :target: https://travis-ci.org/hypothesis/h
-   :alt: Build Status
+.. image:: https://github.com/hypothesis/h/workflows/Continuous%20integration/badge.svg?branch=master
+   :target: https://github.com/hypothesis/h/actions?query=branch%3Amaster
+   :alt:
 .. image:: https://img.shields.io/badge/IRC-%23hypothes.is-blue.svg
    :target: `#hypothes.is`_
    :alt: #hypothes.is IRC channel
```

Comment:
We should remove the IRC channel reference here as it has been deprecated in favor of Hypothesis Open Slack.

Reply:
Done